CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.1.0 - 2018-11-16
+
+### Enhancements
+
+- Allowed `Avatar` to fall back to `initials` when the image fails to load ([#557](https://github.com/Shopify/polaris-react/pull/557))
```

Comment:
_Technically_ permissive language is discouraged, but changing the tense here was a good call.